winterfell/services/userService.js

```diff
@@ -77,3 +79,74 @@ module.exports.upsertUserValues = function(userId, keyValues, callback) {
     userValue.save(callback);
   });
 };
+
+/**
+ * Expected input object from idme:
+ * {
+ *  id
+ *  verified
+ *  affiliation
+ *  fname
+ *  lname
+ *  zip
+ *  uuid
+ *  email
+ * }
+ */
+module.exports.createNewUserFromIdMe = function(idmeInfo, callback) {
+  var newUser = {
+    firstname: idmeInfo.fname,
+    middlename: null,
+    lastname: idmeInfo.lname,
+    email: idmeInfo.email,
+    password: "fakepassword", // todo: generate a short random 8-char password
```

Comment:
Hmm, why cant we just leave the password null? I think they could definitely make a username/password if they want, in addition to oAuth, but I guess having a static default value for password seems a little dangerous, because if we mess up our logic, it would give people access to any oAuth account